gradle.properties

```diff
@@ -1,2 +1,2 @@
 org.gradle.jvmargs=-Xmx1g
-version=0.0.1-SNAPSHOT
+version=0.0.2-SNAPSHOT
```

Comment:
don't think we really need to update the version, it's snapshot not expected to be stable